prototyping/api_tables.ipynb

```diff
@@ -0,0 +1,613 @@
+   "source": [
+    "# Relevant data is found in two different URLs\n",
+    "urls = {\n",
+    "    'api': 'https://api.mtga.untapped.gg/api/v1/',\n",
+    "    'json': 'https://mtgajson.untapped.gg/v1/latest/',\n",
+    "}\n",
+    "\n",
+    "# We read four JSONs in total. The following dictionary facilitates access to its endpoints.\n",
+    "# json: (url, endpoint) \n",
+    "endpoints = {\n",
+    "    'active': ('api', 'meta-periods/active'),\n",
+    "    'analytics': ('api', 'analytics/query/card_stats_by_archetype_event_and_scope_free/ALL?MetaPeriodId='),\n",
+    "    'cards': ('json', 'cards.json'),\n",
+    "    'text': ('json', 'loc_en.json'),\n",
+    "}\n",
+    "\n",
+    "# Header information\n",
+    "headers = {\n",
+    "    'authority': 'api.mtga.untapped.gg',\n",
+    "    'accept': '*/*',\n",
+    "    'accept-language': 'en-US,en;q=0.9,pt;q=0.8',\n",
+    "    'if-none-match': '\"047066ff947f01e9e609ca4cf0d6c0a6\"',\n",
+    "    'origin': 'https://mtga.untapped.gg',\n",
+    "    'referer': 'https://mtga.untapped.gg/',\n",
+    "    'sec-ch-ua': '\"Google Chrome\";v=\"111\", \"Not(A:Brand\";v=\"8\", \"Chromium\";v=\"111\"',\n",
+    "    'sec-ch-ua-mobile': '?0',\n",
+    "    'sec-ch-ua-platform': '\"Windows\"',\n",
+    "    'sec-fetch-dest': 'empty',\n",
+    "    'sec-fetch-mode': 'cors',\n",
+    "    'sec-fetch-site': 'same-site',\n",
+    "    'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36',\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@lru_cache\n",
+    "def request(keyword, send_headers=True, format=''):\n",
+    "    'Returns JSON from the corresponding keyword'\n",
+    "    url_kw, endpoint = endpoints[keyword]\n",
+    "    url = urls[url_kw]\n",
+    "    \n",
+    "    sleep(1) # :)\n",
+    "    if send_headers:\n",
+    "        return requests.get(url+endpoint+format, headers).json()\n",
+    "    else:\n",
+    "        return requests.get(url+endpoint+format).json()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Active JSON\n",
+    "\n",
+    "We need to find what's the current active standard format to compose the analytics request.\n",
+    "We also get all the legal set codes contained in this format."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 38,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "355 ['MID', 'VOW', 'NEO', 'SNC', 'DMU', 'BRO', 'ONE', 'MOM']\n"
+     ]
+    }
+   ],
+   "source": [
+    "def request_active():\n",
+    "    'Returns format ID and lists of standard legal sets'\n",
+    "    latest = dict()\n",
+    "\n",
+    "    # Extracting information from the latest standard BO1 format\n",
+    "    for format in request('active'):\n",
```

Comment:
Good suggestion.  I wrapping `reversed`  around `request` because I'm interested in the last event named 'Ladder' not the first. For some reason, they list obsolete iterations of the event, but they are guaranteed to be ordered from the older to the newest.